packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -191,41 +100,33 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
             children,
             onChange,
             selectedValues,
+            style,
             "aria-disabled": ariaDisabled, // WB-535 avoids passing this prop to the opener
             /* eslint-enable no-unused-vars */
             ...sharedProps
         } = this.props;
-        const {open} = this.state;
-
-        const items = this.getMenuItems();
-
-        const opener = (
-            <ActionMenuOpener
-                {...sharedProps}
-                disabled={items.length === 0 || disabled}
-                onOpenChanged={this.handleOpenChanged}
-                open={open}
-                ref={this.handleOpenerRef}
-                testId={testId}
-            >
-                {menuText}
-            </ActionMenuOpener>
-        );
 
+        const menuItems = React.Children.toArray(this.props.children);
         return (
-            <DropdownCore
+            <Dropdown
                 alignment={alignment}
+                menuItems={menuItems}
+                onChange={this.props.onChange}
```

Comment:
Here you could use the destructured version: `onChange={onChange}`. Same with `selectedValues`.

Reply:
I felt that was unnecessary because those consts would only be used once

Reply:
Friendly ping.